next-door-nuns/requirements.txt

```diff
@@ -0,0 +1,29 @@
+altgraph==0.17
+certifi==2019.9.11
+chardet==3.0.4
+docutils==0.15.2
+entrypoints==0.3
+flake8==3.7.9
+future==0.18.2
+idna==2.8
+Kivy==1.11.1
+kivy-deps.angle==0.1.9
+kivy-deps.glew==0.1.12
+kivy-deps.gstreamer==0.1.17
+kivy-deps.sdl2==0.1.22
+Kivy-examples==1.11.1
+Kivy-Garden==0.1.4
+mccabe==0.6.1
+pefile==2019.4.18
+pycodestyle==2.5.0
+pyflakes==2.1.1
+pygame==1.9.6
+Pygments==2.4.2
+PyInstaller==3.6
+pypiwin32==223
+pywin32==226
+pywin32-ctypes==0.2.0
+requests==2.22.0

```

Comment:
I don't really understand why the `requirements.txt` is so large. Judging from your imports, you actually only have a single dependency, namely `kivy` itself. I don't think any of the other dependencies listed here are necessary, and furthermore, this set of dependency does not resolve properly with these strictly pinned versions.

Here's the error I get when trying to resolve these dependencies:
```py
An error occurred while installing pyinstaller==3.6 --hash=sha256:3730fa80d088f8bb7084d32480eb87cbb4ddb64123363763cf8f2a1378c1c4b7! Will try again.
Installing initially failed dependencies…
     -------------------------------- 0/1 - 00:00:00
Fatal Python error: initsite: Failed to import the site module
Traceback (most recent call last):
  File "c:\users\lmn\.virtualenvs\next-door-nuns--olahcwu\lib\site.py", line 773, in <module>
    main()
  File "c:\users\lmn\.virtualenvs\next-door-nuns--olahcwu\lib\site.py", line 750, in main
    paths_in_sys = addsitepackages(paths_in_sys)
  File "c:\users\lmn\.virtualenvs\next-door-nuns--olahcwu\lib\site.py", line 280, in addsitepackages
    addsitedir(sitedir, known_paths)
  File "c:\users\lmn\.virtualenvs\next-door-nuns--olahcwu\lib\site.py", line 211, in addsitedir
    addpackage(sitedir, name, known_paths)
  File "c:\users\lmn\.virtualenvs\next-door-nuns--olahcwu\lib\site.py", line 179, in addpackage
    exec(line)
  File "<string>", line 1, in <module>
  File "c:\users\lmn\.virtualenvs\next-door-nuns--olahcwu\lib\site-packages\win32\lib\pywin32_bootstrap.py", line 14, in <module>
    for maybe in site.getsitepackages():
AttributeError: module 'site' has no attribute 'getsitepackages'
```

I haven't bothered digging into exactly why this happened, because in truth, you only depend on kivy. So instead, I'll be committing a change into the root repo where your requirements file is shortened down to a single dependency. That way, it should install for everyone.